great_expectations/render/renderer/site_index_page_renderer.py

```diff
@@ -310,6 +317,6 @@ def render(cls, index_links_dict):
 
         except Exception as e:
             logger.error("Exception occurred during data docs rendering: ", e, exc_info=True)
-
-
-
+            exception_traceback = traceback.format_exc()
+            exception_message = f'{type(e).__name__}: "{str(e)}".  Traceback: "{exception_traceback}".'
+            raise GreatExpectationsError(exception_message)
```

Comment:
Again. I'm uncertain about this re-raise.

Reply:
Same motivation as above.